appveyor.yml

```diff
@@ -6,7 +6,7 @@ environment:
         - PYTHON: "C:\\Python35-x64"
 
 install:
-    - ps: Install-Product node 6
+    - ps: Install-Product node 12
```

Comment:
Unrelated: at some point, we should probably replace Travis + AppVeyor here with GitHub Actions.